packages/nimbus/src/components/select/select.mdx

```diff
@@ -12,159 +12,443 @@ menu:
-          )}
-        </Select.Options>
-      </Select.Root>
-    </Box>
-  );
-}
+const App = () => (
+  <Select.Root placeholder="Select a fruit" aria-label="Select a fruit">
+    <Select.Options>
+      <Select.Option>Apples</Select.Option>
+      <Select.Option>Bananas</Select.Option>
+      <Select.Option>Oranges</Select.Option>
+      <Select.Option>Cherries</Select.Option>
+    </Select.Options>
+  </Select.Root>  
+)
 ```
 
-## Props
+## Guidelines
+
+Design guidelines offer a set of standardized rules and recommendations that
+ensure consistency, usability, and accessibility across all design elements and
+interactions.
+
+### Best practice
+
+- **Include a label:** Give the users context on what is being asked for when
+  making a selection. A picker without a label is ambiguous and not accessible.
+  Use “Select” or “Search and select” as a default placeholder label wording.  
+  - In rare cases where context is sufficient to not need a label, these
+    placements should still be given an aria-label in HTML. 
+  - **Placeholders:** Should function as a prompt not a value. Do not use a
+    placeholder as a label.   
+- **Keep contents concise:** Long menu items that wrap to multiple lines should
+  be kept to a minimum. If this becomes a frequent concern, consider an
+  alternate UI component or pattern to allow for more breathing room for
+  content.
+- **Organize content:** The default organization is alphabetical, if a user
+  benefits from better organization please improve their experience with better
+  categories. 
+- **Include help text when needed:** If more information is needed to provide
+  context, include helper text below the field for the user. Replace helper text
+  with error text if applicable and needed.
+- **Don't neglect ARIA attributes:** If you are building a custom select input,
+  ensure that you use proper ARIA attributes to make the select box accessible
+  to screen readers.
+
+#### When to use a solid vs. ghost styled select input
+
+| **Type**    | **Use this instead** |
+|-------------|----------------------|
+| **Solid-style** | <ul><li>The default option to use in forms.</li><li>Typically the best experience to use with content created by a user.</li><li>When you want to include search for the content.</li></ul>                |
+| **Ghost-style** | <ul><li>The default option to use to filter or build rules around content. Ex: filtering assets for the user.</li><li>Best used within rule building experiences such as discounts.</li><li>Ideal to be used in condensed areas, with condensed and familiar options for the user, as in simple filters or quick actions like “delete”.</li></ul> |
+
+### Usage
+
+Select inputs are common and users are familiar with their uses. Keeping
+consistent with the guidelines will help our products intuitive and easy to
+understand, even through complicated areas. 
+
+### Solid-styled widths
+
+These placements are more regular, they have either a maximum width of 268px
```

Comment:
same thing about also mentioning the token and that `268px` is not a tokenized value